frontend/src/screens/Library.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { Image, TouchableHighlight, StyleSheet, ScrollView } from 'react-native'
+import { Appbar, Card } from 'react-native-paper'
+
+import { capitalize } from '../../helpers'
+import { authContext } from '../providers/auth.provider'
+import { authFetch } from '../services/fetch.service'
+
+interface PlaylistProps {
+  title: string
+  tracks: number
+  image: string
+}
+
+const Library = ({ navigation, route }): JSX.Element => {
+  const [playlistItems, setPlaylistItems] = useState<PlaylistProps[]>([])
+  const { user } = useContext(authContext)
+
+  let playlistSearchApi = 'https://api.spotify.com/v1/me/playlists'
+
+  const handleGetPlaylists = () => {
+    if (!user?.id) return
+
+    authFetch(playlistSearchApi)
+      .then((res) => res.json())
+      .then((data) => {
+        setPlaylistItems(
```

Comment:
Ik heb deze functie veranderd, korter en simpeler gemaakt zodat het makkelijker te begrijpen is. Ook hebben we geen fetch meer nodig want we kunnen nu AuthFetch gebruiken. dat is gewoon fetch zonder dat je die tokens en headers mee moet geven